aws/table_aws_ec2_client_vpn_endpoint.go

```diff
@@ -0,0 +1,310 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+
+	ec2v1 "github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEC2ClientVPNEndpoint(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ec2_client_vpn_endpoint",
+		Description: "AWS EC2 Client VPN Endpoint",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("client_vpn_endpoint_id"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ValidationError", "InvalidQueryParameter", "InvalidParameterValue", "InvalidClientVpnEndpointId.NotFound"}),
```

Comment:
Should we ignore `ValidationError`, `InvalidQueryParameter`, `InvalidParameterValue `?